kvmagent/kvmagent/plugins/zrm_plugin.py

```
@@ -0,0 +1,1838 @@
+
+            # Step C: restore original mirror speed (best-effort, failure is non-fatal)
+            try:
+                restore_req = {
+                    http.REQUEST_BODY: json.dumps({
+                        "vmUuid": vm_uuid,
+                        "speed": original_speed,
+                        "waitReadyTimeout": 0
+                    })
+                }
+                self._replication_throttle(restore_req)
+            except Exception as restore_ex:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)